gatsby-ssr.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Implement Gatsby's SSR (Server Side Rendering) APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/ssr-apis/
+ */
+
+// You can delete this file if you're not using it
```

Comment:
Thinking we can safely delete this file, maybe.

Reply:
Will do!

Reply:
@MJJBennett resolved with [this commit](https://github.com/ufvcsa/ufvcsa.github.io/pull/16/commits/203538091a46b8ddce896dfc1dfbb03087bfbea9)